docs/rest-api-example.md

```diff
@@ -0,0 +1,263 @@
+
+pprint.pprint(r.json())
+''' =>
+{'next_result_uri': '/v1/sessions/8797639f35e1ed71b734c035e03e98da/jobs/b1719d4ea8f6d41a830d44918cd7e4d0/result/1',
+ 'results': [{'change_flags': [True, True, True],
+              'columns': [{'name': 'a', 'type': 'INT'},
+                          {'name': 'b', 'type': 'STRING'}],
+              'data': [[1, 'Flink'], [2, 'SQL'], [3, 'gateway']],
+              'result_kind': 'SUCCESS_WITH_CONTENT'}]}
+'''
+next_result_uri = r.json()['next_result_uri']
+```
+
+To continue fetching the next part of results, either follows the `next_result_uri`, or simply increase the `token` by 1 and send out the request.
+
+```python3
+# we still increase the token here for consistency, although we're not using it
+token += 1
+r = requests.get(host + next_result_uri, data=json.dumps(data), headers=headers)
+
+pprint.pprint(r.json())
+''' =>
+{'next_result_uri': '/v1/sessions/8797639f35e1ed71b734c035e03e98da/jobs/b1719d4ea8f6d41a830d44918cd7e4d0/result/2',
+ 'results': [{'change_flags': [True, True, True],
+              'columns': [{'name': 'a', 'type': 'INT'},
+                          {'name': 'b', 'type': 'STRING'}],
+              'data': [[4, 'foo'], [5, 'bar'], [6, 'Hi']],
+              'result_kind': 'SUCCESS_WITH_CONTENT'}]}
+'''
+
+token += 1
+r = requests.get(host + '/v1/sessions/' + session_id + '/jobs/' + job_id + '/result/' + str(token), data=json.dumps(data), headers=headers)
+
+pprint.pprint(r.json())
+''' =>
+{'next_result_uri': '/v1/sessions/8797639f35e1ed71b734c035e03e98da/jobs/b1719d4ea8f6d41a830d44918cd7e4d0/result/3',
+ 'results': [{'change_flags': [True],
+              'columns': [{'name': 'a', 'type': 'INT'},
+                          {'name': 'b', 'type': 'STRING'}],
+              'data': [[7, 'Hello']],
+              'result_kind': 'SUCCESS_WITH_CONTENT'}]}
+'''
+```
+
+If you fail to receive the results, just send out the same request again.
+
+```python3
+r = requests.get(host + '/v1/sessions/' + session_id + '/jobs/' + job_id + '/result/' + str(token), data=json.dumps(data), headers=headers)
+
+pprint.pprint(r.json())
+''' =>
+{'next_result_uri': '/v1/sessions/8797639f35e1ed71b734c035e03e98da/jobs/b1719d4ea8f6d41a830d44918cd7e4d0/result/3',
+ 'results': [{'change_flags': [True],
+              'columns': [{'name': 'a', 'type': 'INT'},
+                          {'name': 'b', 'type': 'STRING'}],
+              'data': [[7, 'Hello']],
+              'result_kind': 'SUCCESS_WITH_CONTENT'}]}
+'''
+```
+
+An empty response indicates the end of the results.
+
+```python3
+token += 1
+r = requests.get(host + '/v1/sessions/' + session_id + '/jobs/' + job_id + '/result/' + str(token), data=json.dumps(data), headers=headers)
+
+pprint.pprint(r.json())
+# => {}
+```
+
+## Close the Session
+
+We now close the session and finish our example.
+
+```python3
+r = requests.delete(host + '/v1/sessions/' + session_id, headers=headers)
+
+pprint.pprint(r.json())
+# => {'status': 'CLOSED'}
+```
```

Comment:
I can contribute with a curl guide if you want..I usually use curl to test such services